SDC-seed/PostgreSQL/tableSchemas.txt

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE listings (
+  id SERIAL,
+  listingId varchar(8),
```

Comment:
Postgres also has a 'char' data type for enforcing fixed-length strings - e.g. char(8) instead of varchar(8)